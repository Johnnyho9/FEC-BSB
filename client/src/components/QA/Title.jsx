import React from 'react';

const Title = () => {
  return (
    <div className='qa-title'>QUESTIONS & ANSWERS</div>
  );
};

export default Title;