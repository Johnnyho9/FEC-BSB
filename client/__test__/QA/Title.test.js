import renderer from 'react-test-renderer';
import Title from '../../src/components/QA/Title.jsx';

 it('Verify that title text is correct', () => {
  const component = renderer.create(
    <Title />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree.children[0]).toEqual('QUESTIONS & ANSWERS');
  expect(tree.props.className).toEqual('qa-title');
  expect(tree.type).toEqual('div');
});
