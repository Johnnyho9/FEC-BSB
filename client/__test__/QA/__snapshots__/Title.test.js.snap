// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify that title text is correct 1`] = `
<div
  className="qa-title"
>
  QUESTIONS & ANSWERS
</div>
`;
